### PR TITLE
[CLA] Update tvtmarine-automation.md

### PR DESCRIPTION
**Pull request by davidtranhp**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/cla/corporate/tvtmarine-automation.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ David Tran david.tran@tvtmarine.com https://github.com/tvtma/
 List of contributors:
 
 David Tran david.tran@tvtmarine.com https://github.com/davidtranhp
-Long Do long.do@tvtmarine.com https://github.com/hoanglong87
```
Comment (mart-e) [inline doc/cla/corporate/tvtmarine-automation.md]: as this email has already made changes (869b833618f727710f75ee61eca508bd3128e181), could you keep the line? If this person no longer works in the company, you can move it at the bottom and add a "(up to 2019-..." message

File: doc/cla/corporate/tvtmarine-automation.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ David Tran david.tran@tvtmarine.com https://github.com/tvtma/
...
+Phong Dao phong.dao@tvtmarine.com https://github.com/phongdao
 Hao Hoang hao.hoang@ma.tvtmarine.com https://github.com/hoanghao2001
 Tommy Tran tu.tran@tvtmarine.com https://github.com/tutran81
-Emily Ha emily.ha@tvtmarine.com https://github.com/emilyha
```
Comment (mart-e) [inline doc/cla/corporate/tvtmarine-automation.md]: this one is fine as never made changes

File: doc/cla/corporate/tvtmarine-automation.md (hunk 1 above)
Comment (davidtranhp) [reply doc/cla/corporate/tvtmarine-automation.md]: Done!

**Comment by mart-e** [comment]
Thanks !

@robodoo r+

**Comment by KangOl** [comment]
@robodoo p=1

**Comment by KangOl** [comment]
@robodoo p=0

**Pull request closed without merging**